chall04/vtran.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+#python 3.6.9
+
+import re
+import sys
+
+def exitIfNotValidInput(siz, arr):
+    i = siz - 1
+    while i >= 0:
+        if (len(arr[i]) != siz) :
+            print("input not a grid")
+            sys.exit()
```

Comment:
You can pass the print inside the sys.exit()

```python
sys.exit("message")
```

https://docs.python.org/2/library/sys.html#sys.exit